pkg/client/config.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright © 2021 Cisco and/or its affiliates. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package client
+
+import (
+	"os"
+)
+
+const (
```

Comment:
The reason why I decided no to do that to keep the dependencies for this lib as minimal as possible. Since the configuration is not complex (so far), I do not think we need external lib for managing configuration.

Reply:
Sounds reasonable. If there were that many config options, we would start to use one anyways.